client/js/_author/components/assessments/_new_assessment.jsx

```diff
@@ -73,7 +73,7 @@ export class NewAssessment extends React.Component {
       <div>
         <Heading
           view="assessments"
-          publishAssessment={this.props.publishAssessment}
+          editOrPublishAssessment={this.props.publishAssessment}
```

Comment:
We should remove this and then remove isRequired in assessments/navigation_bar_content.jsx